### PR TITLE
Add Contact form and Footer

### PR DESCRIPTION
**Pull request by waiyankyaw961999**
### Add contact form and Footer
---

- [x] Add a contact form section
- [x] Add a footer section
- [x] Add a style for contact form using grid 
- [x] Add media query

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -166,10 +160,7 @@ <h2 class="">Multi-Post Stories Gain + Glory</h2>
               </ul>
             </div>
             <div class="flex justify-center">
-              <button
-                class="text-center btn text-white"
-                type="button"
-              >
+              <button class="text-center btn text-white" type="button">
                 See Project
               </button>
```
Comment (Umuhan) [inline index.html:167]: - [x] Please use hover link in the button.

File: index.html [hunk 2]
```diff
@@ -352,5 +330,83 @@ <h2 class="text-center">Skills</h2>
...
+
+    <section class="contact-section relative">
+      <div class="contact-header">
+        <h2 class="text-center">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+      <div class="contact-form">
+        <form action="#" class="flex flex-col">
```
Comment (kit0-0) [inline index.html]: kindly use an action to collect the data
`<form action="https://formspree.io/f/{form_id}" method="post">`

File: index.html [hunk 3]
```diff
@@ -352,5 +330,83 @@ <h2 class="text-center">Skills</h2>
...
+          <input
+            maxlength="30"
+            class="contact-input text-dark"
+            name="name"
+            type="text"
+            placeholder="Full name"
+            required
+          />
```
Comment (Umuhan) [inline index.html]: - [ ] Please indecate the the action here.

File: index.html (hunk 2 above)
Comment (malikhaiderkhan) [inline index.html]: Please use the action attribute that where the data is going after submitting the form.
``` <form action="https://...." method="post/get">```

File: styles/contact.css
Comment (malikhaiderkhan) [inline styles/contact.css:1]: Kindly use the button interactions as it is the requirement of the project.

File: index.html [hunk 4]
```diff
@@ -352,5 +332,83 @@ <h2 class="text-center">Skills</h2>
...
+
+          <input
+            maxlength="30"
```
Comment (victorokoroji) [inline index.html]: - [ ] Kindly remove the max length validation that you added to your email input which is not a requirement for this project. 

Note: some users can have long emails 😄

File: index.html [hunk 5]
```diff
@@ -352,5 +332,83 @@ <h2 class="text-center">Skills</h2>
...
+            type="text-area"
+            class="contact-input message-box"
+            required
+            name="message"
+            placeholder="Hey I'd like to connect to..."
+          ></textarea>
+
+          <button type="submit" class="bg-primary text-center">
+            <span class="text-white">Get in touch</span>
+          </button>
```
Comment (victorokoroji) [inline index.html:373]: - [ ] Kindly implement your button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template.

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @waiyankyaw961999, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- Form submitted successfully ✅ 
- The PR title and description are professional and descriptive ✅ 
- Good UI design ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hi team @waiyankyaw961999, @mohamedgamaleldin999999, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉 
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Very nice design. 💯
✅ Incorporates feedback from previous code reviews.
✅ Pull request has a descriptive title and summary.

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**